bundle/libraries/local_path.go

```diff
@@ -29,21 +27,49 @@ func IsLocalPath(p string) bool {
 		return true
 	}
 
-	// If the path has another scheme, it's a remote path.
-	if isRemoteStorageScheme(p) {
+	if IsRemotePath(p) {
 		return false
 	}
 
-	// If path starts with /, it's a remote absolute path
+	// If the path is absolute, it's a remote path.
 	return !path.IsAbs(p)
```

Comment:
This check is duplicated in `IsRemotePath`. Can we return `!IsRemotePath` instead?